cluster/addons/fluentd-gcp/fluentd-gcp-image/run.sh

```diff
@@ -17,6 +17,12 @@
 # For systems without journald
 mkdir -p /var/log/journal
 
+if [ ! -z "`ls /host/lib/libsystemd* 2>/dev/null`" ]
```

Comment:
Thanks for fixing this this!
You could also just look for existence of `/host/lib/libsystemd.so` file specifically (instead of using a wildcard).